src/utils/wallet.ts

```diff
@@ -540,6 +540,70 @@ const wallet = {
       walletFlags: 0,
     };
   },
+
+  /**
+   * Change the encryption pin on the fields that are encrypted using the pin.
+   * Will not save the access data, only return the new access data.
+   *
+   * @param {IWalletAccessData} accessData The current access data encrypted with `oldPin`.
+   * @param {string} oldPin Used to decrypt the old access data.
+   * @param {string} newPin Encrypt the fields with this pin.
+   * @returns {IWalletAccessData} The access data with fields encrypted with `newPin`.
+   */
+  changeEncryptionPin(accessData: IWalletAccessData, oldPin: string, newPin: string): IWalletAccessData {
```

Comment:
I will have to add it on the other PR when i make the rebase since it does not exist here yet

Reply:
Why haven't you merged the other PR yet?

Reply:
Because it requires a test on this PR to reach the required coverage, I could duplicate the tests and deal with the conflicts, but since the method internals change in this PR it seemed strange making a test for a method that would change its behavior immediately.